scheduling/airflow.sh

```diff
@@ -4,7 +4,5 @@
 unset PYSPARK_DRIVER_PYTHON
 
 # Clone, install, and run
-git clone https://github.com/mozilla/python_etl.git python_etl
-cd python_etl
-pip install .
+pip install git+https://github.com/mozilla/python_mozetl.git
```

Comment:
that implies that the repo is renamed, which GitHub handles automatically with redirects both for the website as well as the Git repo